--- origsrc/akonadi-1.12.1/libs/xdgbasedirs.cpp	2014-04-07 16:02:16.000000000 -0500
+++ src/akonadi-1.12.1/libs/xdgbasedirs.cpp	2014-09-22 00:25:08.841772600 -0500
@@ -337,7 +337,7 @@ QString XdgBaseDirs::findPluginFile( con
 {
   const QStringList searchDirs = findPluginDirs() + searchPath;
 
-#if defined(Q_OS_WIN) //krazy:exclude=cpp
+#if defined(Q_OS_WIN) || defined(Q_OS_CYGWIN) //krazy:exclude=cpp
   const QString pluginName = relPath + QLatin1String( ".dll" );
 #else
   const QString pluginName = relPath + QLatin1String( ".so" );
